v3/src/components/case-table/use-rows.ts

```diff
@@ -55,7 +55,7 @@ export const useRows = (data?: IDataSet) => {
             const strValue = data.getValue(caseId, attr.id)
             const numValue = data.getNumeric(caseId, attr.id)
             const formatStr = attr.format || kDefaultFormatStr
-            const formatted = (numValue != null) && isFinite(numValue) ? format(formatStr)(numValue) : strValue;
+            const formatted = (numValue != null) && isFinite(numValue) ? format(formatStr)(numValue) : strValue
```

Comment:
FYI, the way the semicolon got in there was that semicolons can be required depending on how the next line begins. That used to be true here, but then the code for the subsequent line was changed without removing the semicolon.